src/index.js

```diff
@@ -396,13 +396,17 @@ export class Carousel {
 
 		const {_mask, el, id, pages} = this;
 		const {paginationTemplate, paginationClassName, paginationLabel, paginationTitle} = _options;
-		const pagination = __render(paginationTemplate, {
-			pages,
-			controls: id,
-			className: paginationClassName,
-			label: paginationLabel,
-			title: paginationTitle,
-		});
+		if (pages.length > 1) {
+			const pagination = __render(hasPagination, {
```

Comment:
The render function expects a template. You're passing the setting if the pagination feature should be enabled.

```suggestion
			const pagination = __render(paginationTempate, {
```